lib/drivers/abstract/attribute.js

```diff
@@ -19,12 +19,14 @@ function findJsType(type, defaultDataType) {
 
   switch (dataType.toLowerCase().split('(')[0]) {
     case 'bigint':
+    // case 'mediumint':
```

Comment:
Why comment these lines? It may cause dataType error when init model automatically. eg. `Post.wordCount` jsType will be String

Reply:
```suggestion
    // case 'mediumint':
    case 'mediumint':
```

Reply:
It was commented out to test, looks like my test cases didn't cover this. Let me take a look